rasa/nlu/test.py

```diff
@@ -985,13 +1102,10 @@ def find_intersecting_entities(token: Token, entities: List[Dict]) -> List[Dict]
 
 
 def pick_best_entity_fit(
-    token: Token,
-    candidates: List[Dict[Text, Any]],
-    attribute_key: Text = ENTITY_ATTRIBUTE_TYPE,
-) -> Text:
+    token: Token, candidates: List[Dict[Text, Any]]
+) -> Optional[Dict[Text, Any]]:
```

Comment:
Same question here, since we're changing the function's signature.

Reply:
Same as above. Most likely not used.